### DISCUSSION TITLE
Latest perlnavigator binary does not show symbols

### DISCUSSION THREAD

**Discussion by Ekopalypse** [bscan/PerlNavigator#126] [Q&A] [answered]
Using the version 0.8.12 I'm no longer able to see symbols. The response I get is

JsonError received: Request textDocument/documentSymbol failed with message: File 'C:\\\\**\\\\PerlNavigator\\\\server\\\\perl.tmLanguage.json' was not included into executable at compilation stage. Please recompile adding it as asset or script.

Note sure if this is an issue or whether there is some perl configuration on my side to be done.

**Comment by bscan** [comment, marked as answer]
Oops, yes this was a bug. The 0.8.x series was moved to use webpack and an explicit list of assets for bundling. This resulted in some issues with making sure everything was correctly bundled and pointing to the correct place. I forgot to include the perl.tmLanguage.json file, which is used as part of the shallow parser for building the symbol view.  Thank you for the heads up. This is now fixed in 0.8.14 with this commit: https://github.com/bscan/PerlNavigator/commit/c5051c486989d6df44af889afe2ccd587b471ad3

**Reply by Ekopalypse** [reply]
Thx for the quick fix, 0.8.14 works.
